### PR TITLE
deactivated the perform update checks by default

### PR DESCRIPTION
**Pull request by AviGawande**
This PR solves and close the issue #137 and here is the end result of PR on my local dev server.

### CODE REVIEW

File: src/pretix/base/settings.py [hunk 1]
```diff
@@ -2146,8 +2146,9 @@ def unserialize(cls, s):
         'default': '',
         'type': str
     },
+    # here is the default setting for the updates check 
     'update_check_perform': {
-        'default': 'True',
+        'default': 'False',
```
Comment (norbusan) [inline src/pretix/base/settings.py:2151]: That looks like the correct fix, but what is all the rest for?

Comment (AviGawande) [reply src/pretix/base/settings.py:2151]: that's true rest of the code i had previously commited those change then i figured this above code snippet, that's why

Comment (norbusan) [reply src/pretix/base/settings.py:2151]: THen please remove the changes if they are not necessary nor related.

Comment (AviGawande) [reply src/pretix/base/settings.py:2151]: ok will remove and update the PR

**Review by AviGawande — COMMENTED** [review]
updated the PR branch with the required changes and solves the issue #137

File: src/pretix/base/settings.py (hunk 1 above)
Comment (norbusan) [reply src/pretix/base/settings.py:2151]: Thanks, a bit better now. Please see further comments down here.

File: src/pretix/control/forms/global_settings.py [hunk 2]
```diff
@@ -70,9 +70,12 @@ def __init__(self, *args, **kwargs):
         self.fields['banner_message_detail'].widget.attrs['rows'] = '3'
 
 
+
+
 class UpdateSettingsForm(SettingsForm):
     update_check_perform = forms.BooleanField(
         required=False,
+        initial=False,  # Set the initial value to False
```
Comment (norbusan) [inline src/pretix/control/forms/global_settings.py]: Is this necessary when the above is changed?

File: src/pretix/control/forms/global_settings.py [hunk 3]
```diff
@@ -91,3 +94,4 @@ class UpdateSettingsForm(SettingsForm):
     def __init__(self, *args, **kwargs):
         self.obj = GlobalSettingsObject()
         super().__init__(*args, obj=self.obj, **kwargs)
+
```
Comment (norbusan) [inline src/pretix/control/forms/global_settings.py]: Avoid adding spurious empty lines, since it makes syncing with other patches more difficult.
So please revert this.

File: src/pretix/control/templates/pretixcontrol/global_update.html [hunk 4]
```diff
@@ -25,9 +25,9 @@
             <div class="alert alert-danger">
                 {% trans "The last update check was not successful." %}
                 {% if gs.settings.update_check_result.error == "http_error" %}
-                    {% trans "The pretix.eu server returned an error code." %}
+                    {% trans "The eventyay server returned an error code." %}
                 {% elif gs.settings.update_check_result.error == "unavailable" %}
-                    {% trans "The pretix.eu server could not be reached." %}
+                    {% trans "The eventyay server could not be reached." %}
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/global_update.html:30]: This is a good change, but should be in a different PR.
But ok, we can leave it hear as extra bonus ;-)

File: src/pretix/control/views/global_settings.py [hunk 5]
```diff
@@ -32,7 +32,7 @@ def form_invalid(self, form):
     def get_success_url(self):
         return reverse('control:global.settings')
 
-
+# for UpdateCheckView  implementation
```
Comment (norbusan) [inline src/pretix/control/views/global_settings.py]: Revert please, not needed

File: src/pretix/control/views/global_settings.py [hunk 6]
```diff
@@ -60,7 +60,7 @@ def get_context_data(self, **kwargs):
         return ctx
 
     def get_success_url(self):
-        return reverse('control:global.update')
+        return reverse('control:global.update')
```
Comment (norbusan) [inline src/pretix/control/views/global_settings.py]: Revert please, purely whitespace change. Makes merging patches more complicated in the future.

Comment (AviGawande) [reply src/pretix/control/views/global_settings.py]: ok will revert those changes and update the branch

Comment (AviGawande) [reply src/pretix/control/views/global_settings.py]: but amazed that an extra space can also cause conflicts.

Comment (norbusan) [reply src/pretix/control/views/global_settings.py]: patch does compare the lines as they are there and the lines as saved in the patch file.
If there is a difference, even wrt whitespace, patch/git has extra options to ignore whitespace, but that is dangerous, since Python is white space depending language

**Comment by norbusan** [comment]
I pushed changes that fixed the whitespace issue.

FYI, the best way to do this, 
```
git checkout development src/pretix/control/views/global_settings.py
git checkout development src/pretix/control/forms/global_settings.py
git commit -m "Fix whitespace issues"
```

**Review by norbusan — APPROVED** [review]

**Comment by norbusan** [comment]
Merged the branch, thanks

File: src/pretix/control/views/global_settings.py (hunk 6 above)
Comment (AviGawande) [reply src/pretix/control/views/global_settings.py]: ok,will always keep next time.
